dvc/remote/gdrive.py

```diff
@@ -21,21 +21,27 @@
 
 
 class GDrivePathNotFound(DvcException):
-    def __init__(self, path_info):
-        super().__init__("Google Drive path '{}' not found.".format(path_info))
+    def __init__(self, path_info, hint):
+        hint = "" if hint is None else " {}".format(hint)
+        super().__init__(
+            "GDrive path '{}' not found.{}".format(path_info, hint)
+        )
 
 
-class GDriveAccessTokenRefreshError(DvcException):
-    def __init__(self):
-        super().__init__("Google Drive access token refreshment is failed.")
+class GDriveAuthError(DvcException):
+    def __init__(self, cred_location):
 
+        if cred_location:
+            message = (
+                "GDrive remote auth failed with credentials in '{}'.\n"
+                "Backup first, remove of fix them, and run the command.\n"
```

Comment:
What command? I'm not sure it's clear/obvious.

Reply:
changed to `run DVC again` to make it extra clear

Reply:
👍